rustls/Cargo.toml

```diff
@@ -81,9 +83,9 @@ log = { workspace = true, optional = true }
 # in the error result from `crypto::CryptoProvider::set_default()`, which does not seem
 # worth using the `critical-section` feature with its no-std installation requirements.
 # TODO: specify `once_cell` version in single place - as RECENTLY UPDATED in upstream RUSTLS
-# (It may be best to specify the `once_cell` features here, as may be needed for updates moving forward.)
-once_cell = { version = "1.20.3", default-features = false, features = ["alloc", "race"] }
-# EXPLICIT `portable-atomic` dependency REQUIRED for `unsafe-assume-single-core` feature
+# (keeping `once_cell` features list here, with features as needed by this main crate)
+once_cell = { version = "1.20.3", default-features = false, features = ["alloc", "portable-atomic", "race"] }
```

Comment:
possible TODO comment that `once_cell` with `portable-atomic` should be optional, ideally with reference to an issue number